docs/sources/administration/service-accounts/create-service-account.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I am not sure if there is any advise against this, but I think it would be good to include couple of screenshots to make is visually clear how to do it. For example like [we did it here](https://grafana.com/docs/grafana/latest/enterprise/access-control/manage-role-assignments/manage-user-role-assignments/).

Reply:
I am generally not in favor of adding screenshots unless it adds clarity that cannot otherwise be communicated with the text. From what I've seen, the UI is very simple, so I'm not sure if adding a screenshot adds any real value. It also adds to tech debt when the UI changes (which it will, at some point). In the manage users link that you reference, that screenshot existed previously so I kept it there when I refactored the content. In that example, I should really remove that image. (noting down on to-do list!)